### PR TITLE
Add Material Tooltip

### PR DESCRIPTION
**Pull request by nshahan**
* Material Ripple:
   * Add GPU acceleration.
   * `rippleBindings` have been removed as they are no longer used.
 * Internal updates for compatibility with Angular 3.0.0-alpha.
 * Material Expansion Panel:
   * Fix CSS rule that causes header text to turn gray on hover/focus.
   * Support auto-focus on a content element when the material expansion panel expands.
   * Fix Yes/No button ordering.
 * Material Input:
   * Add a blur update value accessor.
   * Add multiple attribute.
   * Remove unused properties: rows and maxRows.
 * Material Input Multiline: Add auto grow in size.
 * Material Popup: Update change detection for OnPush.
 * Material Progress: Update to animate when main thread is blocked.
 * Material Radio: Adjust size to 24px.
 * Material Yes/No: Add toggle for yes button visibility.
 * Scorecard: Update change detection.
 * Fix flipped alignment positions when isRtl is used.
 * Fix popup event handling.
 * Remove 'uninitialized' as a default value.
 * Remove unused CSS rules.
 * Update styles to meet Material UI spec.
 * Bug fixes.
 * Strong Mode fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
-**Preview Release**
+# AngularDart Components
```
Comment (kevmoo) [inline README.md]: Don't add this header. It's redundant it the places we care about: GitHub and pub site

File: pubspec.yaml [hunk 2]
```diff
@@ -8,9 +8,10 @@ author: Dart Team <misc@dartlang.org>
 environment:
   sdk: '>=1.20.0 <2.0.0'
 dependencies:
-  angular2: '>=2.2.0 <2.3.0'
+  angular2: 3.0.0-alpha
   intl: ^0.14.0
   js: ^0.6.1
+  meta: ^1.0.4
   observable: ^0.14.0+1
```
Comment (kevmoo) [inline pubspec.yaml]: Should make this `>=0.14.0+1 <0.21.0` – I think we just have deprecations, but nothing seems broken.

File: lib/src/laminate/popup/src/popup_size_provider.dart [hunk 3]
```diff
@@ -39,3 +39,22 @@ class PercentagePopupSizeProvider implements PopupSizeProvider {
   num getMaxHeight(num _, num viewportHeight) =>
       viewportHeight * _maxHeightPercentage;
 }
+
+/// Provides a popup size with a fixed maximum width and height in pixels.
+///
+/// A null value for max width or max height indicates no limit.
+class FixedPopupSizeProvider implements PopupSizeProvider {
```
Comment (zoechi) [inline lib/src/laminate/popup/src/popup_size_provider.dart]: Isn't that name misleading? 
The name says `FixedPopupSize` but it seems to be a `MaxSizePopupSizeProvider`. 
What if you want to add a toaster component and need a popup with fixed size for that?

Comment (kevmoo) [inline lib/src/laminate/popup/src/popup_size_provider.dart]: @TedSander thoughts?

Comment (tsander) [inline lib/src/laminate/popup/src/popup_size_provider.dart]: Maybe, but not worth the refactor.

**Pull request merged**